Update 02_connect_cloud_account.mdx (#2541)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx

Error: Please request Azure AD Global Administrator or Privileged Role Administrator to grant admin consent permissions for Service Principal hello-s(77bbxxxx-xxxx-xxxx-xxxx-xxxxxxxx7c54)
```

> The Azure AD Global Administrator or Privileged Role Administrator role can grant admin consent from Azure **App registrations** at [Azure portal](https://portal.azure.com/), then select **API permissions** to grant admin consent.

Reviewer: ```suggestion
To grant admin consent,  a user with the Azure AD Global Administrator or Privileged Role Administrator role goes to [App registrations](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) in the Azure portal and selects **API permissions**.
```
